backend/db/analysis.py

```diff
@@ -249,7 +249,7 @@ def cli():
     if d['counts_update']:
         note = d['note'].strip() if d['note'] else None
         if not note:
-            print('Error: Must provide a --note when using --counts-update.', file=sys.stderr)
+            note = input("Please provide a note: ")
```

Comment:
Looks like this was pretty simple!

I just noticed an issue / some more changes I'd like to request:
1. After the `if not note` block, add the following line: `note = note.strip()`. This change is semi-trivial / quality of life; will remove extra whitespace the user might accidentally enter, e.g. converting `  hello   world  ` to `hello world`.
2. Line 250 here (`note = d['note'].strip() if d['note'] else None`) can now be removed 
3. You can remove the `else` on line 253, putting `counts_update(note, d['schema'])` on the line right after what you did in '1'.
4. Can you run it and see if it allows you to just hit 'enter' (entering nothing) when it asks you for a note? I think probably it will allow that. If so, then right after line 252 (`input(...)`) , let's keep the error there. In fact, let's actually change it to an explicit error rather than a print: `raise ValueError('Must provide a note when using --counts-update.')